### PR TITLE
Fixed formatting of README

### PR DESCRIPTION
**Pull request by svick**
Fixed formatting of headings in README and CONTRIBUTING.

Also moved Microsoft Open Source Code of Conduct to the bottom of README, I don't think it should be at the very top, even before the intro.

This PR is against `master`, not `live`, because that's what users see when they visit this repo.

### CODE REVIEW

**Comment by svick** [comment]
> README.md - There are localized MSDN links in the text. Use non-localized versions.

Should I also fix this?

**Comment by dend** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[CONTRIBUTING.md](https://github.com/svick/visualfsharpdocs/blob/readme/CONTRIBUTING.md) | :white_check_mark:Succeeded | |
[README.md](https://github.com/svick/visualfsharpdocs/blob/readme/README.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C5%5C23%5C2de48aa7-d89a-b079-f888-888d32c75daf%5CPullRequest%5C201705231746100125-291%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: README.md [hunk 1]
```diff
@@ -1,14 +1,13 @@
-
-## Microsoft Open Source Code of Conduct
-
-This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/).
-For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
-
 # Visual F# Documentation
 
 [![Project Status: Suspended - Initial development has started, but there has not yet been a stable, usable release; work has been stopped for the time being but the author(s) intend on resuming work.](http://www.repostatus.org/badges/latest/suspended.svg)](http://www.repostatus.org/#suspended)
```
Comment (svick) [inline README.md]: Is this correct? Is "Suspended" really the official status of F# documentation?

Comment (cartermp) [inline README.md]: This is the official reference documentation for FSharp.Core.  All the conceptual content, including the language reference, is moved over to docs.microsoft.com.  We're still taking contributions here, but only for updating FSharp.Core reference material.

Comment (svick) [inline README.md]: Ok, thanks for the explanation.

Comment (cartermp) [reply README.md]: I'll merge this regardless. Going to likely clean this up and explore manually merging things around.

**Comment by dend** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[README.md](https://github.com/svick/visualfsharpdocs/blob/readme/README.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C6%5C20%5Cd6a0e3a6-9e9b-b4c0-7059-c20dbf9462d1%5CPullRequest%5C201806201535226177-291%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=hu6EsXxGjqSFggl8NdXckibmJvbCraAlXwZ8ZrSZT5w%3D&st=2018-06-20T15%3A31%3A11Z&se=2018-07-21T15%3A36%3A11Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**